src/dotnet/commands/dotnet-list/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Tools.List.ProjectToProjectReferences;
+
+namespace Microsoft.DotNet.Tools.List
+{
+    public class ListCommand : DispatchCommand
+    {
+        protected override string HelpText => @".NET List Command
+
+Usage: dotnet list [options] <object> <command> [[--] <arg>...]]
+
+Options:
+  -h|--help  Show help information
+
+Arguments:
+  <object>   The object of the operation. If a project file is not specified, it defaults to the current directory.
```

Comment:
@blackdwarf this needs some wordsmithing. <object> is pretty obtuse and I don't know what I'd do with it had I not been in the design conversation.

```
  <project>   The path to the project whose <listElement> to list. If a project file is not specified, it defaults to the current directory.
```